src/components/CreateNonFungiblePage/FileUpload.tsx

```diff
@@ -171,3 +185,88 @@ export default function FileUpload() {
     </Flex>
   );
 }
+
+export function CsvFileUpload() {
+  const state = useSelector(s => s.createNftCsvImport);
+  const dispatch = useDispatch();
+  const disclosure = useDisclosure();
+  const [, setLocation] = useLocation();
+
+  const onDrop = useCallback(
+    (files: File[]) => {
+      dispatch(
+        readFileAsDataUrlAction({ ns: 'createNftCsvImport', file: files[0] })
+      );
+    },
+    [dispatch]
+  );
+
+  const { getRootProps, getInputProps } = useDropzone({
+    onDrop,
+    maxFiles: 1,
+    maxSize: 30 * 1024 * 1024,
+    accept: ['text/csv']
```

Comment:
On windows/chrome, the file type is unknown for a csv file, so this filter prevents it from working (I had to comment this out):

![image](https://user-images.githubusercontent.com/703880/119725782-fbae9980-be35-11eb-9cf6-97cde0155dc0.png)

Reply:
Fixed in #431 (disabled file filter)